client/package.json

```diff
@@ -49,7 +49,9 @@
     "@tanstack/react-query": "^4.27.0",
     "@tmcw/togeojson": "^5.7.0",
     "axios": "^1.3.5",
+    "babel-plugin-module-resolver": "^5.0.0",
```

Comment:
not sure we need this library?

Reply:
removed that out, but on my local it gave error that this module was not installed